komorebi/src/windows_api.rs

```diff
@@ -571,6 +575,29 @@ impl WindowsApi {
         Self::set_cursor_pos(rect.left + (rect.right / 2), rect.top + (rect.bottom / 2))
     }
 
+    pub fn get_taskbar_handles() -> Result<(HWND, HWND)> {
+        let taskbar = unsafe { FindWindowA(s!("Shell_TrayWnd"), PCSTR::null()) };
+        let startmenu = unsafe { FindWindowExA(taskbar, HWND { 0: 0 }, s!("Button"), s!("Start")) };
+        Ok((HWND(taskbar.process()?), HWND(startmenu.process()?)))
+    }
+
+    pub fn is_taskbar_hidden(taskbar_hwnd: HWND) -> bool {
+        unsafe { IsWindowVisible(taskbar_hwnd) }.as_bool()
+    }
+
+    pub fn hide_taskbar(taskbar_hwnds: (HWND, HWND), hide: bool) {
+        unsafe {
+            if hide {
+                // 0 hides the taskbar and 5 enables visibility
+                ShowWindow(taskbar_hwnds.0, SHOW_WINDOW_CMD(0));
+                ShowWindow(taskbar_hwnds.1, SHOW_WINDOW_CMD(0));
+            } else {
+                ShowWindow(taskbar_hwnds.0, SHOW_WINDOW_CMD(5));
```

Comment:
Let's replace this with the constant from `windows-rs` https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/UI/WindowsAndMessaging/constant.SW_SHOW.html